data/blog/2024-01-29-vector-announcement.mdx

```diff
@@ -0,0 +1,143 @@
+
+
+#### Similarity Functions
+
+Vector similarity functions measure how "close" or "similar" two vectors are to each other, helping us understand relationships and patterns within the data. Each of these functions yields distinct query results, catering to specific use cases. 
+
+Here are the three supported similarity functions:
+
+- `Cosine Similarity`: Measures the cosine of the angle between two vectors. It is particularly useful when the magnitude of the vectors is not essential, and the focus is on the orientation.
+
+- `Euclidean Distance`: Measures the straight-line distance between two vectors in the multi-dimensional space. It is well-suited for scenarios where the magnitude of vectors is crucial, providing a measure of their spatial separation.
+
+- `Dot Product`: Measures the similarity by multiplying the corresponding components of two vectors and summing the results. 
+
+You can read more information about them and their main use	cases in our docs: [
+Vector Similarity Functions](https://upstash.com/docs/vector/overall/features#vector-similarity-functions)
+
+#### What about related metadata?
+
+Upstah Vectors allows storing `JSON` metadata alongside the vector embeddings. So you can insert embedding-related metadata and get them back while querying the database. Upstash Vector currently supports storing and retrieving metadata alongside vectors. 
+However, it doesn't yet allow filtering based on that metadata. Filtering is in our roadmap and it will enable you to refine vector search results even further. 
+
+See [Metadata](https://upstash.com/docs/vector/overall/features#metadata) docs and [Roadmap](https://upstash.com/docs/vector/overall/roadmap) for more information.
+
+#### Where will be your data stored?
+
+Initially, Upstash Vector will be deployed on two AWS regions; `us-east-1` (N. Virginia)  and `eu-west-1` (Ireland). 
+We are planning to add more regions (and maybe more cloud providers) in the upcoming days.
+
+#### Multitenancy
+
+As a serverless database, Upstash Vector is a multi-tenant service too. We already have very good experience in providing multitenant services on multiple cloud providers and regions. See our [Redis](https://upstash.com/redis) and [Kafka](https://upstash.com/kafka) offerings. 
+
+Similarly, the Upstash vector is deployed on multiple instances in each region, and databases are load-balanced among them due to their size, usage, and traffic. To avoid noisy neighbor problems, we implemented strict CPU parallelism and memory mapping restrictions for each database. 
+
+#### Plans & Limits & Quotas
+
+Unfortunately, every service has its limitations. For Upstash Vector, we have different-sized plans and each plan has its own limits & quotas; such as database size, number of daily requests, max vector dimensions, etc. 
+
+See the Upstash Vector [pricing & plans](https://upstash.com/pricing/vector) page for more information.
```

Comment:
Can we put little pricing info here? otherwise we will lose readers, as they will go to pricing page to learn the prices. I would add something like (please feel free to change to adopt your language): 
```
We will have free, pay-as-you-go, fixed and pro plans. Each plan comes with different limits. Pay-as-you go plan is pure serverless with $0.4 per 100K requests while fixed plan is $60 per month.
```